planner/PolyTraj arc
planner/PolyTraj head
planner/PolyTraj tails
